src/Raven.Studio/typescript/components/pages/database/tasks/ongoingTasks/AddNewOngoingTask.tsx

```diff
@@ -0,0 +1,204 @@
+                    />
+                </Col>
+                <Col>
+                    <div className="flex-grow">
+                        <div className="small-label ms-1 mb-1">Search by name</div>
+                        <div className="clearable-input">
+                            <Form.Control
+                                type="text"
+                                accessKey="/"
+                                placeholder="e.g. Embeddings Generation"
+                                title="Filter tasks"
+                                className="filtering-input"
+                                value={searchText}
+                                onChange={(e) => setSearchText(e.target.value)}
+                            />
+                            {searchText && (
+                                <div className="clear-button">
+                                    <Button variant="secondary" size="sm" onClick={() => setSearchText("")}>
+                                        <Icon icon="clear" margin="m-0" />
+                                    </Button>
+                                </div>
+                            )}
+                        </div>
+                    </div>
+                </Col>
+            </Row>
+            <Row className="mb-4">
+                {filteredTasks.length > 0 ? (
+                    filteredTasks.map((category, index) => (
+                        <div className="mb-4" key={index}>
+                            <HrHeader>
+                                <Icon icon={category.categoryIcon} />
+                                {category.categoryName}
+                            </HrHeader>
+                            <Row className="d-flex row-gap-2">
+                                {category.tasks.map((task, idx) => (
+                                    <Col md={4} key={idx}>
+                                        <TaskItem {...task} />
+                                    </Col>
+                                ))}
+                            </Row>
+                        </div>
+                    ))
+                ) : (
+                    <div className="mt-4 text-center">
+                        <div className="lead text-muted">
+                            <Icon icon="empty-set" size="xl" className="d-block mx-auto mb-2" />
+                            No tasks match your filter criteria
+                        </div>
+                    </div>
+                )}
+            </Row>
+        </div>
+    );
+}
+
+type TaskCardVariant = "Replication" | "Backups" | "Subscriptions" | "ETL" | "Sink";
+
+export interface TaskItemProps {
+    title: string;
+    description: string;
+    iconName: IconName;
+    variant: TaskCardVariant;
+    link: string;
+    target: string;
+    disabled?: boolean;
+    disableReason?: ReactNode;
+    licenseBadge?: LicenseBadgeText;
+    showLicenseBadge?: boolean;
+
+    counterBadge?: null | ReactNode;
```

Comment:
we don't use strict mode
```suggestion
    counterBadge?: ReactNode;
```